docs/ui.md

```diff
@@ -4,7 +4,7 @@ The UI consists of base HTML templates served by the Go App Engine app in `webap
 
 ## UI Principles
 
-- The dashboard should not surface any overall metrics that compare complete runs of different browsers against each other.
+- The dashboard should not surface any overall metrics that compare aligned runs of different browsers against each other.
```

Comment:
This instance of "complete" was actually not about the same sha alignment concept, but rather about summarizing a full run into a single metric to compare browsers by.